default-fields/0000-default-fields.md

```diff
@@ -6,19 +6,224 @@
 # Summary
 [summary]: #summary
 
-One para explanation of the feature.
+Allow struct definitions to supply default values for individual fields,
+and then allow those fields to be omitted from construction:
+
+```rust
+struct Foo {
+    a: &'static str,
+    b: bool = true,
+    c: i32
+}
+
+let foo = Foo {
+    a: "Hello",
+    c: 42
+}
+```
 
 # Motivation
 [motivation]: #motivation
 
-Why are we doing this? What use cases does it support? What is the expected outcome?
+Today, Rust allows you to create an instance of a struct using a literal syntax:
+
+```rust
+struct Foo {
+    a: &'static str,
+    b: bool,
+    c: i32
+}
+
+let foo = Foo {
+    a: "Hello",
+    b: true,
+    c: 42
+};
+```
+
+This requires all fields in the struct be assigned a value, unless the struct implements
+the Default trait.
```

Comment:
This doesn't rely on the Default trait, you can supply any struct instance after the `..` and the new struct takes its values.